examples/structured_data/time_series_anomaly_using_autoencoder.py

```diff
@@ -0,0 +1,323 @@
+"""
+Title: Timeseries anomaly detection using an Autoencoder
+Author: [pavithrasv](https://github.com/pavithrasv)
+Date created: 2020/05/31
+Last modified: 2020/05/31
+Description: Detect anomalies in a timeseries using an Autoencoder.
+"""
+
+"""
+## Introduction
+
+This script demonstrates how you can use a reconstruction convolutional autoencoder model
+to detect anomalies in timeseries data.
+"""
+
+"""
+## Setup
+"""
+
+import io
+import matplotlib
+import numpy as np
+import os
+import pandas as pd
+import tensorflow as tf
+
+from datetime import datetime
+from matplotlib import pyplot as plt
+from matplotlib import dates as md
+from numpy.random import seed
+from tensorflow.keras import layers
+from tensorflow.keras import Sequential
+
+"""
+## Load the data
+
+We will use the [Numenta Anomaly Benchmark(NAB)](https://www.kaggle.com/boltzmannbrain/nab) dataset. It provides artifical timeseries data containing labeled anomalous periods of behavior. Data are ordered, timestamped, single-valued metrics.
```

Comment:
Please shorten text lines to ~80 chars (for version control)

Reply:
Done.